rock_paper_scissors/rps.py

```diff
@@ -2,13 +2,42 @@
 
 import sys
 
+
 def rock_paper_scissors(n):
-  pass 
+    plays = ['rock', 'paper', 'scissors']
+    outcomes = []
+
+    # inner recursive function
+    def find_outcomes(n, result):
+        # print('result', result)
+        # print('rounds', n)
+
+        # base case
+        if n == 0:
+            print('result', result)
+            # print('outcomes', outcomes)
+            outcomes.append(result)
+            return
+
+        # recursive case
+        # for play in plays:
+        #     # outcomes.append(result)
+        #     # collect plays * n in a list
+        #     # count down to 0 to hit base case
+        #     find_outcomes(n-1, result + [play])
+        [find_outcomes(n-1, result + [play]) for play in plays]
```

Comment:
Woah! Great work finding this pattern necessary to approach the base case and using a List Comprehension to build up the values.